docs/authentication.md

```diff
@@ -0,0 +1,44 @@
+## Authentication for Management SDKs
+
+Management SDKs are used to created instances. 
+For those instances we can use OAuth.
+
+All Management SDK's are working with Red Hat Single Sign On (sso.redhat.com) server.
+Authentication to services can be done using Keycloak.js library.
+
+When authenticating please use following keycloak client configuration:
+```
+{
+  "realm": "redhat-external",
+  "auth-server-url": "https://sso.redhat.com/auth/",
+  "ssl-required": "all",
+  "resource": "cloud-services",
```

Comment:
Is `resource` client ID? Because this is different for every client consuming the APIs - for example, the KAS-Ui uses `strimzi-ui`, the CLI uses `rhoas-cli-prod` etc.

Reply:
Yes.. This is the weakest point right now. 
We might:

- add cli as example client
- remove this chapter

Reply:
I believe we should use the CLI as an example to provide some basis, or just use a placeholder like "<your-client-id>"